src/main.rs

```diff
@@ -78,6 +80,21 @@ use umode::UmodeTask;
 
 #[panic_handler]
 fn panic(info: &core::panic::PanicInfo) -> ! {
+    if let Some(location) = info.location() {
+        println!(
+            "panic occurred in file '{}' at line {}",
+            location.file(),
+            location.line()
+        );
+    } else {
+        println!("panic occurred but can't get location information...");
+    }
```

Comment:
This information is already printed on the "info" down later.

Reply:
fixed

Reply:
not fixed?